caikit_nlp/modules/text_generation/peft_prompt_tuning.py

```diff
@@ -344,10 +345,21 @@ def train(
                 underpinning the resource will be converted in place to the correct torch dtype.
             silence_progress_bars: bool
                 Silences TQDM progress bars at train time. Default: True.
+            seed: int
+                Integer to be used as random seed for training.
         Returns:
             PeftPromptTuning
                 Instance of this class with tuned prompt vectors.
         """
+        error.value_check(
```

Comment:
this is removed on `main` now as this causes error in runtime. https://github.com/caikit/caikit-nlp/pull/234